manifests/0000_07_cluster-network-operator_01_crd.yaml

```diff
@@ -67,14 +67,27 @@ spec:
               type: array
               items:
                 type: object
-                required: ["type", "name", "rawCNIConfig"]
+                required: ["type", "name"]
```

Comment:
Make this:

    required: ["type", "name"]
    minProperties: 3
    maxProperties: 3

which effectively says "you must specify `type`, `name`, and exactly one other property" (ie, either `rawCNIConfig` or `macVlanConfig` but not both)

Reply:
fixed.